dev/fattest.simplicity/src/com/ibm/ws/fat/util/ShrinkWrapSharedServer.java

```diff
@@ -39,10 +39,11 @@
      * Creates a {@link SharedServer} then runs any methods in testClass annotated with
      * {@link BuildShrinkWrap} and copies the returned Archives to the servers
      * <p>
-     * Methods must be static, have no parameters, return: {@code Archive}, {@codeArchive[]}, {@code List<Archive>} or {@code Map<Archive,String>}.
+     * Methods must be static, have no parameters, return: {@code Archive}, {@codeArchive[]}, {@code List<Archive>}, {@code Map<Archive,String>}.
+     * or {@code Map<Archive,List<String>>}.
      * If {@code Archive}, {@code}List<Archive> or Archive[] is returned the returned values will be placed to the server's
-     * dropins folders. If a map is returned each archive will be placed wherever the
-     * string points too.
```

Comment:
This doc update doesn't seem related to the code change. I assume you're just correcting the existing doc?

Reply:
Correct